### DISCUSSION TITLE
I want block nesting in CSS

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#338] [Wants]
For most of my work I've been using SASS/SCSS or another preprocessor solely for the ability to nest blocks as I find it lends itself to more succinct and readable code. If CSS supported this natively, it would 

- Simplify the build processes
- Reduce the need to create compatibility package for every build tool
- Save countless GBs in downloads as every project needs a big binary.

<hr>

https://webwewant.fyi/wants/5fbbb494ab888d833450685b/

**Comment by Yay295** [comment]
That issue you linked has a label on it under which you can find more issues: https://github.com/w3c/csswg-drafts/labels/css-nesting
